games/xworld3d/maps/XWorldNav.py

```diff
@@ -0,0 +1,41 @@
+from xworld3d_env import XWorld3DEnv
+from py_gflags import get_flag
+import os
+import random
+
+class XWorldNav(XWorld3DEnv):
+    def __init__(self, item_path):
```

Comment:
Will this class name conflict with xworld2d?

Reply:
yes. we need to figure out a way to resolve this conflict without hardcoding paths in c++ files